### ISSUE TITLE
Cannot read property 'provider' of undefined

### ISSUE THREAD

**Issue by Evineit** [eamodio/vscode-gitlens#1150]
Nothing shows up at the commits and branches views when clicked, only an error notification shows up

- GitLens Version: 11.0.0
  
VS Code Version: 1.51.1
OS: Windows_NT x64 10.0.19041

Steps to Reproduce:

1.Open the commit view
Log
[log.txt](https://github.com/eamodio/vscode-gitlens/files/5541362/log.txt)

image
![Code_13_45_53_934](https://user-images.githubusercontent.com/46513310/99155683-bc7ddf80-267f-11eb-9776-4aec7d2cb7df.png)

**Comment by luizwhite** [comment]
Same with me @Evineit
```

**Comment by luizwhite** [comment]
Same with me @Evineit

<img src="https://user-images.githubusercontent.com/4944479/99160907-2b345a80-26cb-11eb-9c2a-7378cd9ae756.png" height="300" />

```
[2020-11-15 01:43:45:198] [2da] CommitsRepositoryNode(gitlens:repository(/home/luizmarques/Development/gostack/nivel-03/gobarber-frontend-web)).onVisibilityChanged — e={"visible":true}
[2020-11-15 01:43:45:198] [2db] CommitsRepositoryNode(gitlens:repository(/home/luizmarques/Development/gostack/nivel-03/gobarber-frontend-web)).ensureSubscription
[2020-11-15 01:43:45:198] [2dc] CommitsRepositoryNode(gitlens:repository(/home/luizmarques/Development/gostack/nivel-03/gobarber-frontend-web)).subscribe
[2020-11-15 01:43:45:198] [2dc] CommitsRepositoryNode(gitlens:repository(/home/luizmarques/Development/gostack/nivel-03/gobarber-frontend-web)).subscribe completed • 0 ms
[2020-11-15 01:43:45:198] [2dd] CommitsRepositoryNode(gitlens:repository(/home/luizmarques/Development/gostack/nivel-03/gobarber-frontend-web)).triggerChange — false
[2020-11-15 01:43:45:198] [2de] CommitsRepositoryNode(gitlens:repository(/home/luizmarques/Development/gostack/nivel-03/gobarber-frontend-web)).triggerChange — false, false
[2020-11-15 01:43:45:198] [2df] CommitsViewNode.triggerChange — false, false
[2020-11-15 01:43:45:198] [2e0] CommitsView.refreshNode — e=CommitsViewNode, t=false, i=false
[2020-11-15 01:43:45:198] [2e1] CommitsViewNode.refresh — false
[2020-11-15 01:43:45:199] [2e1] CommitsViewNode.refresh completed • 0 ms
[2020-11-15 01:43:45:199] [2da] CommitsRepositoryNode(gitlens:repository(/home/luizmarques/Development/gostack/nivel-03/gobarber-frontend-web)).onVisibilityChanged completed • 0 ms
[2020-11-15 01:43:45:199] [2e2] CommitsView.triggerNodeChange — e=CommitsViewNode
[2020-11-15 01:43:45:199] [2e2] CommitsView.triggerNodeChange completed • 0 ms
[2020-11-15 01:43:45:199] [2db] CommitsRepositoryNode(gitlens:repository(/home/luizmarques/Development/gostack/nivel-03/gobarber-frontend-web)).ensureSubscription completed • 1 ms
[2020-11-15 01:43:45:200] [2e0] CommitsView.refreshNode completed • 1 ms
[2020-11-15 01:43:45:200] [2df] CommitsViewNode.triggerChange completed • 1 ms
[2020-11-15 01:43:45:200] [2de] CommitsRepositoryNode(gitlens:repository(/home/luizmarques/Development/gostack/nivel-03/gobarber-frontend-web)).triggerChange completed • 1 ms
[2020-11-15 01:43:45:200] [2dd] CommitsRepositoryNode(gitlens:repository(/home/luizmarques/Development/gostack/nivel-03/gobarber-frontend-web)).triggerChange completed • 2 ms
[2020-11-15 01:43:45:203] [2e3] CommitsRepositoryNode(gitlens:repository(/home/luizmarques/Development/gostack/nivel-03/gobarber-frontend-web)).ensureSubscription
[2020-11-15 01:43:45:203] [2e3] CommitsRepositoryNode(gitlens:repository(/home/luizmarques/Development/gostack/nivel-03/gobarber-frontend-web)).ensureSubscription completed • 0 ms
[2020-11-15 01:43:45:203] [2e4] GitService.getRemotes — e=/home/luizmarques/Development/gostack/nivel-03/gobarber-frontend-web
[2020-11-15 01:43:45:203] [2e5] GitService.getRepository — e=/home/luizmarques/Development/gostack/nivel-03/gobarber-frontend-web
[2020-11-15 01:43:45:205] [2e5] GitService.getRepository returned /home/luizmarques/Development/gostack/nivel-03/gobarber-frontend-web • 1 ms
[2020-11-15 01:43:45:205] [2e4] GitService.getRemotes completed • 1 ms
```

**Comment by InExtremaRes** [comment]
Same here using:

```plain
GitLens: 11.0.0
VSCode Version: 1.51.1
Electron: 9.3.3
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Linux x64 5.9.4-arch1-1
```

The error fires when I open "Branches". It also occurs when I open "Commits" if I'm not in the `master` branch; in `master` the commits are shown.

The stack trace of the error:

```plain
notificationsAlerts.ts:38 TypeError: Cannot read property 'provider' of undefined
    at Function.getHighlanderProviders (/home/daniel/.vscode/extensions/eamodio.gitlens-11.0.0/dist/webpack:/gitlens/src/git/models/remote.ts:18)
    at BranchNode.getTreeItem (/home/daniel/.vscode/extensions/eamodio.gitlens-11.0.0/dist/webpack:/gitlens/src/views/nodes/branchNode.ts:289)
    at /opt/visual-studio-code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:675
    at async Promise.all (/opt/visual-studio-code/resources/app/out/vs/code/electron-browser/workbench/index 1)
    at m.fetchChildrenNodes (/opt/visual-studio-code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:674)
```

**Comment by easyTree** [comment]
Same here - restarted vscode now all GitLens panels are empty apart from File/Line history.

**Comment by crnjakovic** [comment]
Just installed latest GitLens and same thing happens to me. Can't see commits no matter what.

**Comment by Leslie-Wong-H** [comment]
Same Issue. 

Update to Vscode  1.51 automatically with the  latest feature of gitlens. Great now I can directly manipulate gitlens operation in the source control panel. 

However, when I clicked at "commits", "repositories" and "file history" to expand, the following error notification will always appear, "Cannot read property "provider" of undefined". I was at a private GitHub repository then. I changed it to a public GitHub repository and this problem got resolved. No more error notification. 

Maybe the latest gitlens is having some trouble with the GitHub authentication.

**Comment by luizwhite** [comment]
@Leslie-Wong-H even without a remote the error occurs to me

**Comment by fpragana** [comment]
Same issue with me and all my team

**Comment by weiliang903** [comment]
<img width="578" alt="截屏2020-11-16 09 53 32" src="https://user-images.githubusercontent.com/20644345/99204861-e2c38c80-27f1-11eb-950b-dd77bb024d6a.png">

v11.0.0, macOS 10.15.5
same issue for me.
I am so sad.😫
Then, I installed the previous version v10.2.3. It's ok.

**Comment by Ttou** [comment]
same issue

**Comment by gp3098** [comment]
same issue

**Comment by gentig** [comment]
Same issue with the newest version. Installed the older version 10.2.3 and it works.

**Comment by xidianzhangjun** [comment]
same issue:
MBP macOS 10.15.6
vscode 1.51.1

**Comment by WuYufeng233** [comment]
Same issue. Please release the bugfix version ASAP.

**Comment by eamodio** [comment]
Fixed in v11.0.1

**Comment by lanxb** [comment]
@eamodio 



> Fixed in v11.0.1

so fast ,it's work fine now.

**Comment by thezuliano** [comment]
fixed! thanks so much!
